helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/AssignmentMetadataStore.java

```diff
@@ -42,15 +42,15 @@
   private static final String ASSIGNMENT_METADATA_KEY = "ASSIGNMENT_METADATA";
   private static final String BASELINE_TEMPLATE = "/%s/%s/BASELINE";
   private static final String BEST_POSSIBLE_TEMPLATE = "/%s/%s/BEST_POSSIBLE";
-  private static final String BASELINE_KEY = "BASELINE";
-  private static final String BEST_POSSIBLE_KEY = "BEST_POSSIBLE";
+  static final String BASELINE_KEY = "BASELINE";
+  static final String BEST_POSSIBLE_KEY = "BEST_POSSIBLE";
```

Comment:
Please refer to my previous comments, I don't think you need to remove private if the mock class is extending the methods by calling super.method() first.